datalogger/datalogger.proto

```diff
@@ -12,6 +12,16 @@ message StatisticalAggregate {
   optional uint32 stdev = 5;
 }
 
+// Histogram with integer bucket boundaries and counts
+message IntHistogram {
+  // Parallel list of histogram bucket boundaries and counts.
+  // Bucket 0 is the range from [INT32_MIN, bucket[0])
+  // Bucket i is the range from [bucket[i-1], bucket[i])
+  // If there is one more count element than bucket, the last bucket's upper range is INT32_MAX
+  repeated int32 buckets = 1 [(nanopb).max_count = 10];  // exclusive upper limit
+  repeated uint32 counts = 2 [(nanopb).max_count = 11];
```

Comment:
How forwards/backwards compatible are changes to the max_count on repeated fields?
10 dividers / 11 buckets seems like a reasonable number to me, but are we going to have to make a BiggerIntHistogram in the future if 10 dividers isn't enough?